dpgen/generator/run.py

```diff
@@ -831,25 +833,65 @@ def make_model_devi (iter_index,
     work_path = os.path.join(iter_name, model_devi_name)
     create_path(work_path)
     if model_devi_engine == 'calypso':
-        calypso_run_opt_path = os.path.join(work_path,calypso_run_opt_name)
+        _calypso_run_opt_path = os.path.join(work_path,calypso_run_opt_name)
         calypso_model_devi_path = os.path.join(work_path,calypso_model_devi_name)
-        create_path(calypso_run_opt_path)
         create_path(calypso_model_devi_path)
         # run model devi script
         calypso_run_model_devi_script = os.path.join(calypso_model_devi_path,'calypso_run_model_devi.py')
         shutil.copyfile(calypso_run_model_devi_file,calypso_run_model_devi_script)
-        # run confs opt script
-        run_opt_script = os.path.join(calypso_run_opt_path,'calypso_run_opt.py')
-        shutil.copyfile(run_opt_file,run_opt_script)
-        # check outcar script
-        check_outcar_script = os.path.join(calypso_run_opt_path,'check_outcar.py')
-        shutil.copyfile(check_outcar_file,check_outcar_script)
+        # Create work path list
+        caly_run_opt_path = []
```

Comment:
```suggestion
        calypso_run_opt_path = []
```
To improve the readability of the code